--- jediepcserver.py.factory-default	2018-03-02 23:21:07.000000000 +0100
+++ jediepcserver.py	2018-03-02 23:21:46.000000000 +0100
@@ -100,9 +100,11 @@
         return dict(
             # p.get_code(False) should do the job.  But jedi-vim use replace.
             # So follow what jedi-vim does...
-            params=[p.get_code().replace('\n', '') for p in call_def.params],
+            # params=[p.get_code().replace('\n', '') for p in call_def.params],
+			params=[p.name for p in call_def.params],
             index=call_def.index,
-            call_name=call_def.call_name,
+            # call_name=call_def.call_name,
+			call_name=call_def.name,
         )
     else:
         return []  # nil
@@ -176,7 +178,8 @@
 
 
 def defined_names(*args):
-    return list(map(get_names_recursively, jedi.api.defined_names(*args)))
+    # return list(map(get_names_recursively, jedi.api.defined_names(*args)))
+    return list(map(get_names_recursively, jedi.api.names(*args)))
 
 
 def get_module_version(module):
